File: Java/Arquivos.java
import java.io.File;
import java.io.IOException;
import java.util.Scanner;

public class Arquivos {
	public static void main(String[] args) {
		/*
		 * Para manipular arquivos, deve-se criar um objeto File de java.io
		 *
		 * É extremamente recomendável que se manipule os arquivos em blocos de try
		 */
		try {
			File file = new File("nome-do-arquivo");

			file.createNewFile(); // cria um arquivo no sistema, retorna erro se não possível

			/*
			 * Para escrever em arquivos, deve-se criar um objeto FileWriter:
			 */
			FileWriter writer = new FileWriter("nome-do-arquivo");
			// para escrever:
			writer.write("Hello, World! \n");
			// para fechar o arquivo (EXTREMAMENTE IMPORTANTE!):
			writer.close();

			/*
			 * Para apagar arquivos:
			 */
			file.delete();

			/*
			 * Para ler de arquivos, recomenda-se usar o Scanner:
			 */
			 Scanner scanner = new Scanner(file);
			 scanner.nextLine() // retorna próxima linha, perceba que usar isso várias vezes retorna linhas diferentes
			 // LEMBRE-SE DE FECHAR O SCANNER!

		} catch(IOException exception) {
			System.out.println("Um erro ocorreu :(");
		}
	}
}

File: Python3/Arquivos.py
# Python3 - Manipulação de Arquivos

# Para abrir arquivos, use:
    arquivo = open("arquivo", "modo")

# LEMBRE-SE DE FECHÁ-LOS!!!!!!!!
    arquivo.close()

# É possível usar as linhas de arquivos como parâmetros do for

# Há diversos modos de uso:
    "r" # leitura "read"
    "w" # escrita "write". Substitui o conteúdo do arquivo existente
    "x" # escrita. Retorna erro caso o arquivo já exista
    "a" # escrita. Insere dados no final do arquivo. Cria arquivo
    "b" # binário "binary"
    "t" # texto "text". Padrão
    "+" # atualizar, leitura ou escrita

# Comandos:
    open()
        # depois de variável, abre o arquivo no parâmetro, do modo no parâmetro

# Comandos 2:
    .write()
        # escreve o parâmetro no arquivo antes do ponto
    .writelines()
        # recebe um objeto composto e escreve no arquivo
    .close()
        # fecha o arquivo antes do ponto
    .read()
        # para ler o arquivo todo
    .readlines()
        # ler o arquivo linha por linha em uma lista
